packages/core/utils/lib/async.js

```diff
@@ -20,7 +20,24 @@ function pipeAsync(...methods) {
  */
 const mapAsync = curry(pMap);
 
+/**
+ * Because of how mysql works, making parallel requests can cause deadlocks.
+ * This function will run the requests sequentially if mysql is used. Else,
+ * it will encapsulate them in a Promise.all.
+ *
+ * @type { import('./async').MapAsync }
```

Comment:
I don't think that the types matches here

Reply:
even if we are using those same params and passing them to mapAsync? shouldnt they be the same?

Reply:
It is not because MapAsync is a curried function 👍

Reply:
true 🤔  I will change it